playground/vitestSetup.ts

```diff
@@ -249,19 +260,11 @@ export async function startDefaultServe(): Promise<void> {
 }
 
 function startStaticServer(
-  resolved: ResolvedConfig,
-  config?: InlineConfig
+  resolved: ResolvedConfig, // resolved config which load by test root dir
+  config?: InlineConfig // user config which near by test file
```

Comment:
Why are we using both `resolved` and `config`? Is it possible to use `testConfig` in `startDefaultServe` function instead?

Reply:
good. Thanks for tips. 😁